src/openlcb/Defs.hxx

```diff
@@ -246,6 +246,16 @@ struct Defs
     /// station may react to this by restoring locomotive speed settings.
     static constexpr uint64_t CLEAR_EMERGENCY_STOP_EVENT = 0x010000000000FFFCULL;
 
+    /// "Power supply brownout detected below minimum required by node"
+    /// This event can be generated when a node detects that it has
+    /// insufficient power for normal operations.
+    static constexpr uint64_t NODE_POWER_BROWNOUT_EVENT = 0x010000000000FFF1ULL;
+
+    /// "Ident button combination pressed"
+    /// This event can be generated by a node when it is instructed to generate
+    /// an identification event.
+    static constexpr uint64_t NODE_IDENT_BUTTON_EVENT = 0x010000000000FE01ULL;
```

Comment:
This is FE00 in the standard.

Reply:
Oops, will fix in a min. I read it off-by-one line.